app/models.py

```diff
@@ -1,9 +1,25 @@
 import requests
 import os
 from flask import json
-from elasticsearch import Elasticsearch
 
-es = Elasticsearch()
+
+api_url = os.getenv('DM_API_URL')
+api_access_token = os.getenv('DM_BUYER_FRONTEND_API_AUTH_TOKEN')
+search_url = os.getenv('DM_SEARCH_API_URL') + "/search"
+search_access_token = os.getenv('DM_BUYER_FRONTEND_SEARCH_API_AUTH_TOKEN')
+
+if api_access_token is None:
+    print('Token must be supplied in DM_BUYER_FRONTEND_API_AUTH_TOKEN')
+    raise Exception("DM_BUYER_FRONTEND_API_AUTH_TOKEN token is not set")
+if api_url is None:
+    print('API URL must be supplied in DM_API_URL')
+    raise Exception("DM_API_URL is not set")
+if search_access_token is None:
+    print('Token must be supplied in DM_BUYER_FRONTEND_SEARCH_API_AUTH_TOKEN')
+    raise Exception("DM_BUYER_FRONTEND_SEARCH_API_AUTH_TOKEN token is not set")
+if search_url is None:
+    print('Search API URL must be supplied in DM_SEARCH_API_URL')
+    raise Exception("DM_SEARCH_API_URL is not set")
```

Comment:
This is a good plan, I wasn't sure of the best place to do this so put it here.  We'll need to do the same for all frontend apps (which currently check in the same way as this).  Let's do that next in a separate set of pull requests.